docs/ops/movement/Transpose_1.md

```diff
@@ -6,30 +6,26 @@
 
 **Short description**: *Transpose* operation reorders the input tensor dimensions.
 
-**Attributes**:
+**Detailed description**: *Transpose* operation reorders the input tensor dimensions. Source indexes and destination indexes are bound by the formula: 
+\f[output[i(order[0]), i(order[1]), ..., i(order[N-1])] = input[i(0), i(1), ..., i(N-1)]\\ \quad \textrm{where} \quad i(j) \quad\textrm{is in the range} \quad [0, (input.shape[j]-1)]\f]
 
-No attributes available.
+
+**Attributes**: *Transpose* operation has no attributes.
 
 **Inputs**:
 
-* **1**: "arg" - the tensor to be transposed. A tensor of type T1. **Required.**
-* **2**: "input_order" - the permutation to apply to the axes of the input shape. Must be a vector of element T2 type, with shape [n], where n is the rank of "arg". The tensor's value must contain every integer in the range [0,n-1]. If an empty list is specified [] then the axes will be inverted. A tensor of type T2. **Required.**
+* **1**: `arg` - the tensor to be transposed. A tensor of type `T` and arbitrary shape. **Required.**
+* **2**: `input_order` - the permutation to apply to the axes of the first input shape. A 1D tensor of `n` elements `T_AXIS` type and shape `[n]`, where `n` is the rank of the first input or `0`. The tensor's value must contain every integer in the range `[0, n-1]`, but if an empty tensor is specified (shape `[0]`) then the axes will be inverted. **Required.**
 
 **Outputs**:
 
-*   **1**: A tensor with shape and type matching 1st tensor.
+*   **1**: A tensor of type `T` and first input rank shape with transposed first input tensor dimensions.
```

Comment:
The sentence is overcomplicated. Can we write just: `A tensor of type \`T\` and transposed shape according to the rules specified above.`

Reply:
Sure we can:) Thanks, done